build.cmd

```diff
@@ -139,14 +142,28 @@ if not exist %_msbuildexe% echo Error: Could not find MSBuild.exe.  Please see h
 
 :: All set to commence the build
 
-setlocal
+setlocal EnableDelayedExpansion
 if defined __MscorlibOnly goto PerformMScorlibBuild
 
 echo Commencing build of native components for %__BuildOS%.%__BuildArch%.%__BuildType%
 echo.
 
 rem arm64 builds currently use private toolset which has not been released yet
-if /i "%__BuildArch%" =="arm64" goto GenVSSolution
+if /i "%__BuildArch%" == "arm64"  (
+  REM TODO, remove once the toolset is open.
+
+  if /i "%__ToolsetDir%" == "" (
+    echo "A toolset directory is required for the Arm64 Windows build."
+    exit /b 1
+  )
+)
+
+REM TODO, remove once the toolset is open.
+if /i "%__BuildArch%" == "arm64" set PATH=%PATH%;%__ToolsetDir%\cpp\bin
```

Comment:
@gkhanna79 That doesn't work as expected.  The expansion rules for paren blocks cause problems with the path setup (even with the delayed expansion).  vcvars does what @jashook is doing here.